models/cdm/cdm/Native.py

```diff
@@ -91,9 +91,10 @@ def dataLikelihood(self, model, step):
             omegaZ = parameters[omegaZIdx]
 
             # make a source using the sample parameters
-            cdm = source(x=x, y=y, d=d,
+            cdm = source(x=x, y=y, d=d, opening=opening,
```

Comment:
Modified function variable referencing/order for consistency.

Reply:
with python pass-by-name, the order of arguments is not important